packages/driver/src/cypress/error_messages.ts

```diff
@@ -25,11 +25,7 @@ function removeLeadingSlash (str: string) {
   return str.substring(li + 1)
 }
 
-const formatConfigFile = (projectRoot: string, configFile: string | false) => {
-  if (configFile === false) {
-    return '`cypress.config.{ts|js}` (currently disabled by --config-file=false)'
-  }
-
+const formatConfigFile = (projectRoot: string, configFile: string) => {
```

Comment:
Potentially could use `Cypress.Config['configFile']` for a single source of truth for the type of `configFile`, not sure though. I wonder if we want to start working towards this?

Eg - you wouldn't have needed this change if we'd been doing this in the first place, it would be free since you'd be just changing the type for `Cypress.Config['configFile']`.

Reply:
Done in [84bfdcc](https://github.com/cypress-io/cypress/pull/20643/commits/84bfdcc97c31c9acd6077486f9976bb6b18ddf73), also converted the `projectRoot` definition as it's the same deal. I agree, we should use the predefined types when and where we can.